abot/bot.py

```diff
@@ -26,15 +26,33 @@ class Abort(Exception):
 
 class Backend:
     def configure(self, **config):
+        """Feed configuration to the backend before initialization.
+
+        This method can be useful to pass usernames or passwords to the Backend
+        and store/handle them before the Bot initializates the Backend.
+        """
         raise NotImplementedError
 
     async def initialize(self):
+        """Execute one time operation which should run when the Backend starts."""
         raise NotImplementedError()
 
     async def consume(self):
+        """Execute Backend logic."""
```

Comment:
Please make note this is supposed to be an async generator that yields `Event` subtypes

Reply:
Done